package.json

```diff
@@ -15,6 +15,7 @@
     "@types/react": "^18.0.0",
     "@taquito/taquito": "^14.0.0",
     "react": "18.2.0",
+    "relay-runtime": "14.1.0",
```

Comment:
Forcing a resolution here since there's a bad peer dep from graphql-codegen. 

This isn't related but thought I'd fix while I was debugging.